scipy/sparse/tests/test_sputils.py

```diff
@@ -92,3 +92,48 @@ def test_validateaxis(self):
         # These function calls should not raise errors
         for axis in (-2, -1, 0, 1, None):
             func(axis)
+
+    def test_get_index_dtype(self):
+        func = sputils.get_index_dtype
```

Comment:
I appreciate the test case, but I'm not a fan of aliasing to `func` here.

Reply:
all good